core/src/test/java/org/bitcoinj/wallet/WalletTest.java

```diff
@@ -743,10 +744,12 @@ public void isConsistent_spent() {
     }
 
     @Test
+    @Ignore
     public void isTxConsistentReturnsFalseAsExpected() {
         Wallet wallet = Wallet.createDeterministic(UNITTEST, ScriptType.P2PKH);
         TransactionOutput to = createMock(TransactionOutput.class);
         EasyMock.expect(to.isAvailableForSpending()).andReturn(true);
+        // TODO: Now that this is moved to Wallet (TransactionBag) mocking doesn't work here
```

Comment:
This is issue #2

Reply:
I've managed to fix the test by applying this patch:
```
-        EasyMock.expect(to.isMineOrWatched(wallet)).andReturn(true);
+        EasyMock.expect(to.getScriptPubKey()).andReturn(ScriptBuilder.createOutputScript(wallet.currentReceiveAddress()));
```
The idea is that if I convert one of the wallet's addresses to a script and return that in the mocked `getScriptPubKey()`, it should be mostly equivalent to the previous mocked `isMineOrWatched()` returning true.